The trait `std::clone::Clone` is not implemented for `BenchmarkModuleRecord<B>

**Describe the bug**
When running `./run_checks.sh all` on main, an error is returned.

**To Reproduce**
`./run_checks.sh all`

Error is:
```
    Blocking waiting for file lock on build directory
    Finished `dev` profile [unoptimized] target(s) in 4.07s
     Running `target/debug/xtask run-checks all`
[2024-05-27T01:29:24Z INFO  xtask::utils::cargo] cargo install typos-cli --version 1.16.5 --color=always
    
     Ignored package `typos-cli v1.16.5` is already installed, use --force to override
[2024-05-27T01:29:24Z INFO  xtask::runchecks] Running typos check 
    
    
[2024-05-27T01:29:24Z INFO  xtask::runchecks] Cargo: fmt
[2024-05-27T01:29:24Z INFO  xtask::utils::cargo] cargo fmt --check --all -- --color=always
    
[2024-05-27T01:29:25Z INFO  xtask::utils::cargo] cargo clippy --color=always --all-targets -- -D warnings
    
    Checking backend-comparison v0.15.0 (/home/will/code/burn/backend-comparison)
    Checking burn-import v0.15.0 (/home/will/code/burn/crates/burn-import)
   Compiling onnx-tests v0.15.0 (/home/will/code/burn/crates/burn-import/onnx-tests)
   Compiling onnx-inference v0.15.0 (/home/will/code/burn/examples/onnx-inference)
   Compiling pytorch-import v0.15.0 (/home/will/code/burn/examples/pytorch-import)
   Compiling image-classification-web v0.15.0 (/home/will/code/burn/examples/image-classification-web)
    Checking regression v0.15.0 (/home/will/code/burn/examples/simple-regression)
    Checking custom-wgpu-kernel v0.15.0 (/home/will/code/burn/examples/custom-wgpu-kernel)
    Checking named-tensor v0.15.0 (/home/will/code/burn/examples/named-tensor)
    Checking text-generation v0.15.0 (/home/will/code/burn/examples/text-generation)
    Checking custom-image-dataset v0.15.0 (/home/will/code/burn/examples/custom-image-dataset)
    Checking mnist-inference-web v0.15.0 (/home/will/code/burn/examples/mnist-inference-web)
    Checking text-classification v0.15.0 (/home/will/code/burn/examples/text-classification)
    Checking custom-training-loop v0.15.0 (/home/will/code/burn/examples/custom-training-loop)
    Checking guide v0.15.0 (/home/will/code/burn/examples/guide)
    Checking custom-renderer v0.15.0 (/home/will/code/burn/examples/custom-renderer)
error[E0277]: the trait bound `BenchmarkModuleRecord<B>: std::clone::Clone` is not satisfied
   --> backend-comparison/benches/load_record.rs:56:17
    |
56  |     type Args = BenchmarkModuleRecord<B>;
    |                 ^^^^^^^^^^^^^^^^^^^^^^^^ the trait `std::clone::Clone` is not implemented for `BenchmarkModuleRecord<B>`
    |
note: required by a bound in `burn_common::benchmark::Benchmark::Args`
   --> /home/will/code/burn/crates/burn-common/src/benchmark.rs:115:16
    |
115 |     type Args: Clone;
    |                ^^^^^ required by this bound in `Benchmark::Args`

For more information about this error, try `rustc --explain E0277`.
error: could not compile `backend-comparison` (bench "load-record") due to 1 previous error
warning: build failed, waiting for other jobs to finish...
```

**Expected behavior**
Run `./run_checks.sh all` with no errors

**Desktop (please complete the following information):**
 - OS: Ubuntu 20.04
 - `cargo 1.78.0`
 - On `main`, diff: `c7ad25a`

--- a/backend-comparison/benches/load_record.rs
+++ b/backend-comparison/benches/load_record.rs
@@ -53,7 +53,7 @@ struct LoadRecordBenchmark<B: Backend> {
 }
 
 impl<B: Backend> Benchmark for LoadRecordBenchmark<B> {
-    type Args = BenchmarkModuleRecord<B>;
+    type Args = BenchmarkModule<B>;
 
     fn name(&self) -> String {
         format!("load_record_{:?}", self.kind).to_lowercase()
@@ -67,7 +67,9 @@ impl<B: Backend> Benchmark for LoadRecordBenchmark<B> {
         10
     }
 
-    fn execute(&self, record: Self::Args) {
+    fn execute(&self, module: Self::Args) {
+        let record = module.into_record();
+
         let _ = match self.kind {
             Kind::Lazy => {
                 let module = self.config.init(&self.device);
@@ -86,9 +88,8 @@ impl<B: Backend> Benchmark for LoadRecordBenchmark<B> {
     fn prepare(&self) -> Self::Args {
         let module = self.config.init(&self.device);
         // Force sync.
-        let module_initialized = module.clone();
 
-        module_initialized.into_record()
+        module.clone()
     }
 
     fn sync(&self) {

